MIGRATION_GUIDE.md

```diff
@@ -14,13 +14,13 @@ It is noted as a behavior change but in some way it is not; with the previous im
 
 We will consider adding `NOT NULL` back because it can be set by `ALTER COLUMN columnX SET NOT NULL`, but first we want to revisit the whole resource design.
 
-## vX.XX.X -> v0.85.0
+### snowflake_procedure resource changes
+#### *(deprecation)* return_behavior
+`return_behavior` parameter is deprecated because it is also deprecated in the Snowflake API.
 
-### Migration from old (grant) resources to new ones
```

Comment:
why this is removed?

Reply:
it looked like it was re-written? at the top is a new migration guide intro.

Reply:
It was added to point the users to the resource migration guide, I would leave it as it was

Reply:
Yes, please revert this whole paragraph. Also, `TestInt_CreateProcedures` is failing, so we cannot merge it yet.

Reply:
re-added paragraph. Commented out problem in TestInt_CreateProcedures, its an issue I already filed with docs-discuss channel. Included jira ticket in the comment

Reply:
test still failing, but passes locally:

```
--- PASS: TestInt_CreateProcedures (21.14s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Java:_returns_result_data_type (2.72s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Java:_returns_table (2.44s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Javascript (0.30s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Javascript:_no_arguments (0.24s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Scala:_returns_result_data_type (4.02s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Scala:_returns_table (3.95s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Python:_returns_result_data_type (4.89s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_Python:_returns_table (1.81s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_SQL:_returns_result_data_type (0.42s)
    --- PASS: TestInt_CreateProcedures/create_procedure_for_SQL:_returns_table (0.35s)
PASS
```

unsure what is going on.